### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tomasz-dudziec-red**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+simple example app to store results of played match inside company 
+example deployment: https://tomaszd.herokuapp.com
```
Comment (tomaszd) [inline README.md:2]: Thank you!

Comment (tomaszd) [inline README.md:2]: better to start with capital

Comment (tomaszd) [reply README.md:2]: .

**Review by tomaszd — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tomasz-dudziec-red) [reply README.md:2]: ok i will do that

Comment (tomaszd) [reply README.md]: Thanks for adding changes

**Pull request merged**